### PR TITLE
Cycle 4 Funding: Specreduce Project Manager

### PR DESCRIPTION
**Pull request by camipacifici**
In the file

### CODE REVIEW

**Comment by pllim** [comment]
Does "2 years" extend beyond Cycle 4? If so, is that allowed?

Would be nice if you can elaborate how this fits into [Astropy Roadmap](https://github.com/astropy/astropy-project/blob/main/roadmap/roadmap.md) and how it complements what STScI already pays for.

File: finance/proposal-calls/cycle4/specreduce_pm.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+### Title
+
+Specreduce Project Manager
+
+### Project Team
+
+- Tim Pickering, Camilla Pacifici, Clare Shanahan, Kelle Cruz, ...
```
Comment (kbwestfall) [inline finance/proposal-calls/cycle4/specreduce_pm.md:7]: I'd like to be involved as well :)

Comment (camipacifici) [reply finance/proposal-calls/cycle4/specreduce_pm.md]: Added!

**Comment by camipacifici** [comment]
> Does "2 years" extend beyond Cycle 4? If so, is that allowed?

The instructions say "1 to 3 years support".

> Would be nice if you can elaborate how this fits into [Astropy Roadmap](https://github.com/astropy/astropy-project/blob/main/roadmap/roadmap.md) and how it complements what STScI already pays for.

Ok, I added a bit more text, but I'll address the context of the Astropy Roadmap in the next iterations.

Thanks!

**Comment by kelle** [comment]
In the budget, we should specify that we envision this person being an independent contractor. (Right? or is there a possibility we'd need to do a subaward to someone's institution?)

**Comment by mhvk** [comment]
It would seem this contract requires a rather special type of person, who combines good knowledge of spectroscopic reduction with coding skills, since otherwise the amount of supervision required might also be more than the team can muster. Do you have particular people in mind who might be interested?

**Comment by kelle** [comment]
Consider adding a budget range.  What's the minimum amount of funds in the next year that would be useful to meet the goals of this position.

**Comment by kelle** [comment]
> It would seem this contract requires a rather special type of person, who combines good knowledge of spectroscopic reduction with coding skills, since otherwise the amount of supervision required might also be more than the team can muster. Do you have particular people in mind who might be interested?

We have a couple folks in mind that we will encourage to apply, but we do envision an open search.  

I agree that we're looking for a more senior, experienced person with a specific skill set. I've been very pleasantly surprised and impressed with the applications we've received for other searches and I'm reasonable optimistic that we can find someone for this. We should try!

**Comment by camipacifici** [comment]
> Consider adding a budget range. What's the minimum amount of funds in the next year that would be useful to meet the goals of this position.

Done. ~6 hours a week minimum, ~1 full day a week maximum.

**Comment by aaryapatil** [comment]
Just to confirm, the initial contract would be for 1 year, and if it's successful it would be extended to a total of 2 years (1+1)?

**Comment by camipacifici** [comment]
Correct.

**Comment by eteq** [comment]
Please react to this comment to vote on this proposal (👍, 👎, or no reaction for +0)

File: finance/proposal-calls/cycle4/specreduce_pm.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+- build the infrastructure to support flux calibration,
+- include an additional extraction method based on PSF weighting.
+
+The Project Manager we are planning to hire should be a senior software engineer with
+experience in spectroscopy. They will push the project forward by:
+- integrating the efforts of folks based at different istitutions and with different interests
+([JWST data analysis](https://jwst-docs.stsci.edu/jwst-post-pipeline-data-analysis/jwst-data-analysis-software-packages),
+[PypeIt](https://pypeit.readthedocs.io/en/release/),
+[DRAGONS](https://www.gemini.edu/observing/phase-iii/reducing-data/dragons-data-reduction-software), etc.)
+- proposing solutions for the architectural challanges,
```
Comment (pllim) [inline finance/proposal-calls/cycle4/specreduce_pm.md:46]:
```suggestion
- proposing solutions for the architectural challenges,
```

**Comment by eteq** [comment]
Hi Camilla and team,

I'm writing on behalf of Astropy's Finance Committee regarding the outcome of your funding request. We have opened a new issue #415 to track the progress.

We are pleased to be able to let you know that, following consultation with the community, we are able to approve this request. **We can currently fund the Year 1 amount of ${DOLLAR} (US) to carry out the project**.  Funding beyond this amount will be contingent upon the availability of funds. (We will be using the full budgets of all of the approved requests to craft future grant and funding proposals.) We assume you will be using this full year 1 budget, if that is not the case please contact us immediately.

Ana Gabela and I will be your contacts on the Finance Committee to facilitate this award. Please get in touch with us if you have any questions or concerns. Please do not reach out to NumFOCUS directly.

In addition, new to this funding cycle, is the assignment of a COTR (Contracting Officer's Technical Representative) to each funded project. This concept is borrowed from government funding agencies, although it is to be stressed that Astropy's goal is to make the COTR role as low-overhead as possible. The COTR’s primary responsibility is to make sure the work is happening at the expected pace and, if necessary, to be a liaison between the funded project and the Finance Committee or CoCo. The COTR for your project will be assigned shortly and we’ll also be sending out more details about how we see this working.

If your request includes a new hire, please consult our newly compiled [hiring guide](https://docs.google.com/document/d/1xD8qdMW3GhPP86XR1fXE_9Cb03SxNtokgWG1Aoxor54/edit).

The next steps are:

* Please make sure that you are registered with Open Collective (https://opencollective.com): this is the system you will use to submit your invoices for payment. Note that your invoices should be submitted through the   [Astropy NASA](https://opencollective.com/astropy/projects/astropy-nasa) project on Open Collective.
* We will reach out through NumFOCUS, Astropy's Fiscal Sponsor, to set up an “independent contractor agreement” (ICA; effectively a contract) and formal scope of work for the project.
* While this is in progress, the Finance Committee and the Coordination Committee will identify a COTR for each project. More information on this will be in the issue for your project once the COTR is identified.


We expect the work to be carried out between 2024-05-01 and 2025-04-30 (the “period of performance”). If you come to realize that the work will take more time than originally planned, you can apply for an extension following the procedure described at https://github.com/astropy/astropy-project/blob/main/finance/process/request-extension.md.

Congratulations --- we are really looking forward to seeing you put these funds to good use!

Erik Tollerud
on behalf of the Astropy Finance Committee

**Pull request merged**